coreutils: error in tests/ls/acl.sh test

### Nixpkgs version

- Unstable (25.05)

### Steps to reproduce

Starting with nix 2.8.0 installed from debian 12 apt repo on ppc64le.

Coreutils fails to build on both nixpkgs 24.11 and unstable. Log output is below as an example for `nix-shell -p hello` , let me know if there is other output I can provide that would be helpful.

### Can Hydra reproduce this build failure?

Please select the Hydra Status.

### Link to Hydra build job

_No response_

### Relevant log output

```console

ERROR: tests/ls/acl
===================

acl.sh: set-up failure: 
ERROR tests/ls/acl.sh (exit status: 99)

============================================================================
Testsuite summary for GNU coreutils 9.6
============================================================================
# TOTAL: 657
# PASS:  498
# SKIP:  158
# XFAIL: 0
# FAIL:  0
# XPASS: 0
# ERROR: 1
============================================================================
See ./tests/test-suite.log
Please report to bug-coreutils@gnu.org
============================================================================
make[4]: *** [Makefile:23202: tests/test-suite.log] Error 1
make[4]: Leaving directory '/tmp/nix-build-coreutils-9.6.drv-0/coreutils-9.6'
make[3]: *** [Makefile:23310: check-TESTS] Error 2
make[3]: Leaving directory '/tmp/nix-build-coreutils-9.6.drv-0/coreutils-9.6'
make[2]: *** [Makefile:23570: check-am] Error 2
make[2]: Leaving directory '/tmp/nix-build-coreutils-9.6.drv-0/coreutils-9.6'
make[1]: *** [Makefile:23087: check-recursive] Error 1
make[1]: Leaving directory '/tmp/nix-build-coreutils-9.6.drv-0/coreutils-9.6'
make: *** [Makefile:23572: check] Error 2
error: builder for '/nix/store/w3bq87iwv3d0h6j3dnlmz2z2p47s1wy0-coreutils-9.6.drv' failed with exit code 2
error: 1 dependencies of derivation '/nix/store/lavhl68zbl8zfbypv2950b8ihb624kv7-gcc-wrapper-14-20241116.drv' failed to build
error: 1 dependencies of derivation '/nix/store/0c8ixjjql6y99hsa9d3lvwslny01fids-stdenv-linux.drv' failed to build
error: build of '/nix/store/0c8ixjjql6y99hsa9d3lvwslny01fids-stdenv-linux.drv', '/nix/store/0s521ifkjjmag7jai6pzg9nadx4hnq2r-bash-interactive-5.2p37.drv', '/nix/store/55czg44g3hsf4mpsq878f1yn3b1dkfwz-hello-2.12.1.drv' failed
```

### Additional context

_No response_

### System metadata

Cannot run due to coreutils not building.

### Notify maintainers



---

**Note for maintainers:** Please tag this issue in your pull request description. (i.e. `Resolves #ISSUE`.)


### I assert that this issue is relevant for Nixpkgs

- [x] I assert that this is a bug and not a support request.
- [x] I assert that this is not a [duplicate of an existing issue](https://github.com/NixOS/nixpkgs/issues?q=is%3Aopen+is%3Aissue+label%3A%220.kind%3A+build+failure%22). 
- [x] I assert that I have read the [NixOS Code of Conduct](https://github.com/NixOS/.github/blob/master/CODE_OF_CONDUCT.md) and agree to abide by it.

### Is this issue important to you?

Add a :+1: [reaction] to [issues you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[issues you find important]: https://github.com/NixOS/nixpkgs/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc

--- a/pkgs/tools/misc/coreutils/default.nix
+++ b/pkgs/tools/misc/coreutils/default.nix
@@ -97,6 +97,9 @@ stdenv.mkDerivation rec {
 
       # intermittent failures on builders, unknown reason
       sed '2i echo Skipping du basic test && exit 77' -i ./tests/du/basic.sh
+
+      # fails when syscalls related to acl not being available, e.g. in sandboxed environment
+      sed '2i echo Skipping ls -al with acl test && exit 77' -i ./tests/ls/acl.sh
     ''
     + (optionalString (stdenv.hostPlatform.libc == "musl") (
       concatStringsSep "\n" [

